### ISSUE TITLE
[Bug]: web-ui login false

### ISSUE THREAD

**Issue by caihexin** [QuivrHQ/quivr#2362]
### What happened?

![image](https://github.com/QuivrHQ/quivr/assets/13762872/b70aa1fe-93b9-4064-aac4-fffc7d665732)

worker        | [2024-03-19 09:35:00,050: WARNING/ForkPoolWorker-58] last_synced: 2024-03-13T02:08:54+00:00
worker        | [2024-03-19 09:35:00,059: ERROR/ForkPoolWorker-58] Task celery_worker.process_integration_brain_sync[051c99cc-68cf-4e74-8aec-db16d8f6e71f] raised unexpected: ValueError("time data '2024-03-13T02:08:54+00:00' does not match format '%Y-%m-%dT%H:%M:%S.%f%z'")
worker        | Traceback (most recent call last):
worker        |   File "/usr/local/lib/python3.11/site-packages/celery/app/trace.py", line 477, in trace_task
worker        |     R = retval = fun(*args, **kwargs)
worker        |                  ^^^^^^^^^^^^^^^^^^^^
worker        |   File "/usr/local/lib/python3.11/site-packages/celery/app/trace.py", line 760, in __protected_call__
worker        |     return self.run(*args, **kwargs)
worker        |            ^^^^^^^^^^^^^^^^^^^^^^^^^
worker        |   File "/code/celery_worker.py", line 194, in process_integration_brain_sync
worker        |     last_synced = datetime.strptime(
worker        |                   ^^^^^^^^^^^^^^^^^^
worker        |   File "/usr/local/lib/python3.11/_strptime.py", line 568, in _strptime_datetime
worker        |     tt, fraction, gmtoff_fraction = _strptime(data_string, format)
worker        |                                     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
worker        |   File "/usr/local/lib/python3.11/_strptime.py", line 349, in _strptime
worker        |     raise ValueError("time data %r does not match format %r" %
worker        | ValueError: time data '2024-03-13T02:08:54+00:00' does not match format '%Y-%m-%dT%H:%M:%S.%f%z'

**Comment by caihexin** [comment]
I have been trying for three weeks, but I still haven't been able to successfully log in and try the app. Can integrate it into a Docker like Ollama ???

**Comment by caihexin** [comment]
![image](https://github.com/QuivrHQ/quivr/assets/13762872/ce167c8f-5bef-49a5-9967-a7327bc638b5)

**Comment by XiandanErizo** [comment]
same  mark

**Comment by ingeniumdesign** [comment]
you dont load the css/js
i edit in the .env all "`localhost`" in my local IP from the Server. But i cant login with the default user: admin@quir.app and pw: admin

**Comment by eli103** [comment]
Got the same issue,  Did anyone encounter the same and fixed ?

**Comment by asubramania** [comment]
@eli103 I got the same problem today.
Still trying...

**Comment by asubramania** [comment]
I tried the following to get rid unknown service and http connect errors. I don't know this is the best way. But this helped me to proceed testing further..
I am on ubuntu 22.04 running on AWS EC2 instance…

1/. docker compose down

2/. In the project home directory, find and back up your docker-compose.yml file:
cp docker-compose.yml docker-compose.yml-ORIG

3/. Then, do the following:
In docker-compose.yml file, I added these 2 lines at the bottom of "worker" service:
    extra_hosts:
      - "host.docker.internal:host-gateway"

4/. See the "worker" service as an example which is pasted here:

  worker:
    pull_policy: if_not_present
    image: stangirard/quivr-backend-prebuilt:latest
    env_file:
      - .env
    build:
      context: backend
      dockerfile: Dockerfile
    container_name: worker
    command: celery -A celery_worker worker -l info
    restart: always
    depends_on:
      - redis
    extra_hosts:
      - "host.docker.internal:host-gateway"

5/. Similarly, I added these 2 lines at the bottom of all services in docker-compose.yml file.
    extra_hosts:
      - "host.docker.internal:host-gateway"

6/. Then, I did the following:
docker compose up
